features/items.md

```diff
@@ -0,0 +1,237 @@
+
+The item name is the unique name of the item which is used in the .sitemap, .rule etc files. The name must be unique across all item files. The name should only consist of letters, numbers and the underscore character. Spaces cannot be used.
+
+## Label text
+
+The label text has two purposes. First, this text is used to display a description of the specific item (for example, in the sitemap). Second, it can be used to format or transform output from the item (for example, making DateTime output more readable). If you want to display a special character you must mask the character with a '%'. So, to display one '%' enter the text '%%'.
+
+## Formatting
+
+Formatting is done applying [Java formatter class syntax](http://docs.oracle.com/javase/7/docs/api/java/util/Formatter.html#syntax), therefore the syntax is
+
+```xtend
+%[argument_index$][flags][width][.precision]conversion
+```
+
+Only the leading '%' and the trailing 'conversion' are mandatory. The **argument_index$** must be used if you want to convert the value of the item several times within the label text or if the item has more than one value. Look at the DateTime and Call item in the following example.
+
+```xtend
+Number    MyTemperature  "The Temperature is [%.1f] °C"   { someBinding:somevalue }
+String    MyString       "Value: [%s]"                    { someBinding:somevalue }
+DateTime  MyLastUpdate   "Last Update: [%1$ta %1$tR]"     { someBinding:somevalue }
+```
+
+The output would look like this:
+
+    Temperature 23.2 °C
+    Value: Lorem ipsum
+    Last Update: Sun 15:26
+
+## Transforming
+
+Another possibility in label texts is to use a transformation. They are used for example to translate a status into another language or convert technical value into human readable ones. To do this you have to create a .map file in your ```${openhab_home}/conf/transform``` folder. These files are typical key/value pair files.
+
+    key1=value1
+    key2=value2
+    ...
+
+Let's make a small example to illustrate this function. If you have a sensor which returns you the number 0 for a closed window and 1 for an open window, you can transform these values into the words "open" or "closed". Create a map file named window.map for example and add the desired keys and values.
+
+    0=closed
+    1=opened
+    UNDEFINED=unknown
+    -=unknown
+
+Next we define two items. One showing the raw value as it is provided from our sensor and one with transformed value.
+
+```xtend
+Number WindowRaw          "Window is [%d]"                  { someBinding:somevalue }
+Number WindowTransformed  "Window is [MAP(window.map):%s]"  { someBinding:somevalue }
+```
+
+The output will be
+    Window is 1
+    Window is opened
+
+Transform file use ISO-8859-1 encoding. If you want your accent and special character to show in the right way, you need to precise this encoding in your editor. If you must use UTF-8, you will need to escape the character. See [UTF-8 encoding table](http://www.utf8-chartable.de/)
```

Comment:
This is not correct anymore. In fact I'm using all kinds of unicode symbols (even emojis) in my items/sitemap/transformation: https://github.com/ThomDietrich/openhab-config/blob/master/transform/astro.map